<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1215</width>
    <height>751</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1215</width>
    <height>750</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>921</width>
      <height>641</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="settingsBox">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>540</y>
      <width>251</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QCheckBox" name="axisVisibility">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Axis visibility</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="gridVisibility">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Grid visibility</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="returnButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Return to starting position</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="selectFunctionComboBox">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>40</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1215</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
